tinygrad/ops.py

```diff
@@ -31,8 +31,9 @@ def __init__(self, op: Op, src: Tuple[Union[LazyOp, LazyBuffer], ...], arg: Any
     self.op = op
     self.src = src
     self.arg = arg
+    self.buffers = ()
     try:
-      self.buffers = tuple([y for x in src for y in x.buffers])
+      for x in src: self.buffers += x.buffers if isinstance(x, LazyOp) else (x,)
```

Comment:
do you need the isinstance check here? it wasn't there before

Reply:
I don't, this could be `x.buffers`, but having it here allows me to delete the `buffer` method on the `LazyBuffer`.

Reply:
hmm, should probably be part of a separate pr then, just to keep this one as simple as possible